### DISCUSSION TITLE
二聯發票與三聯發票有何差異? 如為做為維修購買證明呢?

### DISCUSSION THREAD

**Discussion by AnAnistired** [1101-GS4719-C/Python-Community-GS4719-C#793] [General]
分享小知識：原來三聯發票是這樣的
[連結](https://www.kocpc.com.tw/archives/464064)

**Comment by Pietra1226** [comment]
感謝分享生活小知識~

**Comment by moonnight3503** [comment]
常常不知道三聯兩聯的差異，反正別人給我什麼就拿什麼，現在終於知道了😂😂，感謝分享！

**Comment by dabbie109303501** [comment]
之前都覺得差不多，原來三聯和二聯發票有差異

**Comment by ching020721** [comment]
因為時常需要跟教授報銷購買的工具，近期常常請商家開三聯式發票，因此也有了解過兩個的差異，這是不錯的小知識感謝分享~
